spec-tests/src/test/java/org/candlepin/spec/HostedTestSpecTest.java

```diff
@@ -56,19 +59,24 @@ void hostedShouldBeAlive() throws ApiException {
         assertThat(isAlive).isTrue();
     }
 
-    @Test
-    void shouldCleanServerData() throws ApiException {
-        hosted.createContent(Content.random());
-        hosted.createContent(Content.random());
-        hosted.createProduct(Products.random());
+    @Nested
+    @Execution(ExecutionMode.SAME_THREAD)
```

Comment:
This is to prevent the hosted cleanup from deleting test data from under other tests running in parallel. I am considering completely deleting it as any use of hosted cleanup will break parallel tests in hosted mode.